lib/top_hangman/game.rb

```diff
@@ -1,5 +1,48 @@
+require_relative 'round'
+
 module TopHangman
   class Game
+    attr_reader :base_words
+
+    def initialize(file_name)
+      @base_words = read_words_file(file_name)
+      @list_of_words_size = base_words&.size
+      @history_of_words = []
+      @round = create_round
+    end
+
+    private
+
+    def read_words_file(file_name)
+      return nil unless File.exist?(file_name)
+
+      base_file = File.open(file_name, 'r')
+      base_file.readlines.map(&:chomp).select { |line| line.size > 5 && line.size < 12 }
```

Comment:
what is 5 and 12? Maybe replace them with constants to avoid magic numbers

Reply:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/26731448/193955870-b0a02d1e-3b71-494b-a1a4-0615c15d7445.png">